### PR TITLE
DX-955 Add test for ci building

### PR DESCRIPTION
**Pull request by NullableInt**

### CODE REVIEW

File: .docker/entrypoint [hunk 1]
```diff
@@ -7,4 +7,8 @@ bundle check || bundle install
 
 echo "Running: $@"
 
-exec $@
+if [[ $@ = "ci" ]]; then
```
Comment (asbjornu) [inline .docker/entrypoint:10]: I think this would be the correct syntax

```suggestion
if [[ "$@" == "ci" ]]; then
```

**Comment by NullableInt** [comment]
No idea what fails in the build, it looks to me as if it pushes it

**Comment by NullableInt** [comment]
Asking it to build again made it pass. I do not like this.

File: .docker/Dockerfile [hunk 2]
```diff
@@ -86,6 +86,7 @@ RUN rm -rf $BUNDLE_HOME/cache
 RUN rm -rf $GEM_HOME/cache
 
 COPY --chown=jekyll:jekyll .docker/entrypoint ${JEKYLL_BIN}/entrypoint
+COPY --chown=jekyll:jekyll .docker/ci-build-publish ${JEKYLL_BIN}/ci-build-publish
```
Comment (asbjornu) [inline .docker/Dockerfile:89]:
```suggestion
COPY --chown=jekyll:jekyll .docker/ci-build-publish ${JEKYLL_BIN}/gh-pages-publish
```

File: .docker/entrypoint [hunk 3]
```diff
@@ -7,4 +7,8 @@ bundle check || bundle install
 
 echo "Running: $@"
 
-exec $@
+if [[ "$@" == "ci" ]]; then
+  exec /usr/jekyll/bin/ci-build-publish
```
Comment (asbjornu) [inline .docker/entrypoint:11]:
```suggestion
  exec /usr/jekyll/bin/gh-pages-publish
```

File: .docker/entrypoint [hunk 4]
```diff
@@ -7,4 +7,8 @@ bundle check || bundle install
 
 echo "Running: $@"
 
-exec $@
+if [[ "$@" == "ci" ]]; then
```
Comment (asbjornu) [inline .docker/entrypoint:10]:
```suggestion
if [[ "$@" == "gh-pages-publish" ]]; then
```

**Comment by NullableInt** [comment]
I think the build "fails" because it's reusing the `latest` tag.

**Pull request merged**